pkg/kube/controllers/extendedjob/errand_reconciler.go

```diff
@@ -117,6 +117,8 @@ func (r *ErrandReconciler) createJob(ctx context.Context, eJob ejv1.ExtendedJob)
 	}
 	template.Labels[ejv1.LabelEJobName] = eJob.Name
 
+	// TODO modify secret references to use latest version of 'versioned secret' for the job
+	// TODO don't create a job if any versioned secret still has v0
```

Comment:
+1 for using latest
What does delaying job creation get us?
Won't we create for v1 when it pops up anyway?